src/local_neighborhood.py

```diff
@@ -62,34 +62,19 @@ def run(network=None, nodes=None, output_file=None, k=None, singularity=False):
         volumes = list()
 
         bind_path, node_file = prepare_volume(nodes, work_dir)
-        print("bind_path: ", bind_path)
-        print("node_file: ", node_file)
-
         volumes.append(bind_path)
-        print("volumes1: ", volumes)
 
         bind_path, network_file = prepare_volume(network, work_dir)
         volumes.append(bind_path)
-        print("volumes2: ", volumes)
-
-        # PathLinker does not provide an argument to set the output directory
-        # Use its --output argument to set the output file prefix to specify an absolute path and prefix
-        out_dir = Path(output_file).parent
-        print("out_dir: ", out_dir)
-        # PathLinker requires that the output directory exist
-        out_dir.mkdir(parents=True, exist_ok=True)
-        bind_path, mapped_out_dir = prepare_volume(str(out_dir), work_dir)
+
+        bind_path, mapped_output_file = prepare_volume(output_file, work_dir)
         volumes.append(bind_path)
-        print("bind_path: ", bind_path)
```

Comment:
Is this fit the output requirement for the LocalNeighborhood?

Reply:
Yes, the way you set up the output file matched my expectations. We'll have to test it to make sure I didn't miss anything.